xpdacq/beamtimeSetup.py

```diff
@@ -38,9 +38,12 @@ def _start_beamtime(PI_last, saf_num, experimenters=[], *,
         src = os.path.join(os.path.dirname(__file__), 'Ni24.D')
         dst = os.path.join(glbl.usrAnalysis_dir, 'Ni.D')
         shutil.copy(src, dst)
-        # import sample
-        # FIXME: leave for user?
-        # import_sample(bt, saf_num)
+        
+        # pre-populated scan plan
+        ScanPlan(bt, ct, 60)
```

Comment:
do we have a ct 5?  I think we use that in the documentation.  I would advocate for a ct 1 also, and possibly a ct "min".  Why?  ct "min" gives people the shortest time that is available.  In this case "min" is 1/frame-rate, i.e., acquisition time.

btw, what will happen if an exposure is requested that is shorter than the 1/frame-rate ?  Do we have exception handling for that case?  The right behavior in general should be to not create it and to throw a warning.  However, the right behavior in the startup sequence should probably be the same, but maybe to pause with a question to the user to make sure that the frame-rate is set correctly.